src/Components/FriendsPage/FriendsPage.js

```diff
@@ -1,41 +1,75 @@
-import React, { useState } from 'react'
-import './FriendsPage.css'
+import React, { useState, useEffect } from 'react';
+import './FriendsPage.css';
+import { getFriends } from '../../ApiCalls';
 
 function FriendsPage() {
+  const [search, setSearch] = useState('');
+  const [friends, setFriends] = useState([]);
+  const [filteredFriends, setFilteredFriends] = useState([]);
+  const [newFriendName, setNewFriendName] = useState('');
 
-  const friends = ['Trevor', 'Gus', 'Joe', 'Jill']
-  const [search, setSearch] = useState('Search Friends');
-  const [filteredFriends, setFilteredFriends] = useState(['Trevor', 'Gus', 'Joe', 'Jill']);
+  useEffect(() => {
+    getFriends(1)
+      .then(data => {
+        setFriends(data.data.friends);
+        setFilteredFriends(data.data.friends);
+      })
+      .catch(err => console.log(`There has been an error: ${err}`))
+  }, []);
 
-  const displayFriends = filteredFriends.map(friend => {
-    return <div className='long-tile'>{friend}</div>
-  })
+  useEffect(() => {
+    filterFriends(search);
+  });
```

Comment:
This seems problematic, thought I'm not entirely sure - with no dependencies listed in the `useEffect` dependency array, this hook would run constantly. If I'm following the logic of how you have this component set up, I think what you could do is add `[search]` as a dependency, so each time that the `search` piece of the form state changes, the `useEffect` reruns and generates a new list of filtered friends. 

Let me know your thoughts and if this suggestion makes sense!